cohere-cookbooks/Semantic search with Cohere embeddings/Wikipedia_Semantic_Search_With_Cohere_Embeddings_Archives.ipynb

```diff
@@ -0,0 +1,554 @@
+      "\n",
+      "As mentioned in the provided information, Turing created the theoretical concept of a \"Turing machine\" in 1936, which was a significant contribution to the emerging field of computing. This imaginary machine introduced the idea of a computer program and set the foundation for modern computing. \n",
+      "\n",
+      "During World War II, Turing played a crucial role in breaking German military codes, most notably those generated by the Enigma machine. His work at Bletchley Park, Britain's code-breaking center, was instrumental in providing Allied forces with crucial intelligence that greatly aided their efforts during the war. \n",
+      "\n",
+      "Despite his remarkable contributions, Turing's life took a tragic turn due to the societal attitudes and legal framework of his time. He was convicted of \"gross inde \n",
+      "\n",
+      "\n",
+      "Result of API call 3 :\n",
+      "Alan Turing was a pioneering British mathematician, computer scientist, and cryptanalyst. He is widely regarded as one of the most influential figures in the development of theoretical computer science and is often referred to as the \"father of computer science and artificial intelligence.\"\n",
+      "\n",
+      "As you mentioned, Turing is known for his concept of the Turing machine, which he introduced in 1936. The Turing machine was a theoretical device that served as a simple model for a general-purpose computer. It could perform any computation if given the appropriate instructions, providing a framework for the design of modern computers.\n",
+      "\n",
+      "During World War II, Turing played a crucial role in breaking German military codes, including those generated by the Enigma machine. His work at Bletchley Park, the central site for British code-breaking operations, significantly contributed to the Allied victory.\n",
+      "\n",
+      "After the war, Turing continued his work in computer science and mathematics. He designed the Automatic Computing Engine (ACE), one of the first designs for a \n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Send the same query to the Command R+ model using the Bedrock converse API but add the results of the semantic search as context\n",
+    "conversation = [\n",
+    "    {\n",
+    "        \"role\": \"user\",\n",
+    "        \"content\": [{\"text\": prompt}],\n",
+    "    }\n",
+    "]\n",
+    "\n",
+    "try:\n",
+    "    for i in range(3):\n",
+    "        print('Result of API call', str(i+1), ':')\n",
+    "        # Send the message to the model, using a basic inference configuration.\n",
+    "        response = bedrock_rt.converse(\n",
+    "            modelId='cohere.command-r-plus-v1:0',\n",
+    "            messages=conversation,\n",
+    "            inferenceConfig={\"maxTokens\": 200, \"temperature\": 0.5, \"topP\": 0.9},\n",
```

Comment:
Should we note why we're using temp = .5

Reply:
Added a note to explain that 0.5 is used to illustrate variance in the response. Linked the Amazon Bedrock User Guide for readers interested in changing these values.